paasta_tools/cli/cmds/boost.py

```diff
@@ -46,8 +46,9 @@ def add_subparser(subparsers):
     boost_parser.add_argument(
         '-c', '--cluster',
         type=str,
+        action='append',
         required=True,
-        help="Cluster to boost. ex: nova-prod. You can find a list of paasta clusters with `paasta list-clusters'",
+        help="Cluster to boost. ex: nova-prod .This option can be specified multiple times.",
```

Comment:
Drop the example cluster or put some dummy not yelpy one to match what we do elsewhere.

Reply:
Can you also make this argument behave similar to -c in other commands like paasta status? We take a comma separated list and have completion support.

Reply:
done